src/create-sync.coffee

```diff
@@ -0,0 +1,94 @@
+_ = require 'underscore'
+Backbone = require 'backbone'
+Backbone.$ = require 'jquery' # TODO remove after upgrading to backbone 1.2+
+
+Utils = require './utils'
+
+
+module.exports = (hostOptions) ->
+  (method, model, options) ->
+    methodMap =
+      create: 'POST'
+      update: 'PUT'
+      patch:  'PATCH'
+      delete: 'DELETE'
+      read:   'GET'
+
+    type = methodMap[method]
+
+    # Default options, unless specified.
+    _.defaults(options || (options = {}), {
+      emulateHTTP: Backbone.emulateHTTP,
+      emulateJSON: Backbone.emulateJSON
+    })
+
+    # Default JSON-request options.
+    params = { type: type, dataType: 'json' }
+
+    # Ensure that we have a URL.
+    unless options.url
```

Comment:
What are the thoughts on leveraging `options.url` for customizing the URL?

Reply:
Wouldn't that need to be provided at every call to `Backbone.sync`?

Reply:
Yeah, and I think it's possible given that we control when this sync method is called. I was thinking it can be defined on every collection. 

But... I don't think it adds much since it is just a different way to accomplish the same thing.